### PR TITLE
Update component “file-inputs”

### PR DESCRIPTION
**Pull request by honeckkaitlynn**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/components/file-inputs.md [hunk 1]
```diff
@@ -1,105 +1,133 @@
...
-![Single-file input anatomy](/images/components/file-input/singlefile_anatomy.svg)
+## Anatomy
 
-The single-file input consists of an action and input field.
+![File Input anatomy](/images/file-input_anatomy.svg)
 
-1. **Icon (optional):** Gives more definition to the upload action.
-2. **Action:** A single action associated with the file input, such as 'Browse' or 'Upload content.'
-3. **Input field:** A clickable container that brings up the file explorer.
+1. **Button:** The shape that encloses the icon and action. When selected, the users default file explorer will open.
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:22]:
```suggestion
1. **Button:** The shape that encloses the icon and action. When selected, the user's default file explorer will open. 
```

File: src/_areas/design-system/components/file-inputs.md [hunk 2]
```diff
@@ -1,105 +1,133 @@
...
+## Anatomy
 
-The single-file input consists of an action and input field.
+![File Input anatomy](/images/file-input_anatomy.svg)
 
-1. **Icon (optional):** Gives more definition to the upload action.
-2. **Action:** A single action associated with the file input, such as 'Browse' or 'Upload content.'
-3. **Input field:** A clickable container that brings up the file explorer.
+1. **Button:** The shape that encloses the icon and action. When selected, the users default file explorer will open. 
+2. **Icon:** Gives more definition to the upload action. Can be used in place of the action text due to lack of space or on mobile.
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:23]:
```suggestion
2. **Icon:** Gives more definition to the upload action. Can be used in place of the label due to lack of space or on mobile. 
```
Should we remain consistent with our button anatomy and call the text a "label"?

File: src/_areas/design-system/components/file-inputs.md [hunk 3]
```diff
@@ -1,105 +1,133 @@
...
 
-The single-file input consists of an action and input field.
+![File Input anatomy](/images/file-input_anatomy.svg)
 
-1. **Icon (optional):** Gives more definition to the upload action.
-2. **Action:** A single action associated with the file input, such as 'Browse' or 'Upload content.'
-3. **Input field:** A clickable container that brings up the file explorer.
+1. **Button:** The shape that encloses the icon and action. When selected, the users default file explorer will open. 
+2. **Icon:** Gives more definition to the upload action. Can be used in place of the action text due to lack of space or on mobile. 
+3. **Action:** A single action associated with the File Input, such as 'Browse' or 'Upload content.'
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:24]: Same here if you rename to label above.

File: src/_areas/design-system/components/file-inputs.md [hunk 4]
```diff
@@ -1,105 +1,133 @@
...
 4. **Input text:** Placeholder text will instruct users to select file(s) or drop file(s) into the dropzone for upload. Once a file has been selected, the filename will appear.
+5. **Input field:** A clickable container that brings up the file explorer.
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:26]: I'm assuming the entire component being clicked will bring up the file explorer. I think 4 and 5 could potentially be combined to just be an input field. The input field contains a placeholder until a file is selected, at which point the file name is displayed.

Then we could have a line below stating the entire component is clickable.

File: src/_areas/design-system/components/file-inputs.md [hunk 5]
```diff
@@ -1,105 +1,133 @@
...
-For detailed documentation on our single-file input anatomy, view the [ZUI single-file input design specs](https://xd.adobe.com/view/f50eaaf7-355c-424d-93d1-59d91ca911a4-a05d/specs/).
+For detailed documentation on our File Input anatomy, view the [design specs](https://xd.adobe.com/view/c3866087-1ec9-47ff-bfa1-2e98a0b23b04-96d5/).
 
----
+- - -
 
 ## Types
 
-We have two types of file inputs that allow the user to upload the appropriate amount of files through their file explorer or drag and drop.
+Currently, we have one type of File Input that allows the user to upload a single file through their file explorer or drag and drop.
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:34]: I'd remove if we have only 1

File: src/_areas/design-system/components/file-inputs.md [hunk 6]
```diff
@@ -1,105 +1,133 @@
...
 
----
+- - -
 
 ## Behavior
 
 ### Form
 
-When the single-file input is used in a form, it should be grouped with related fields and include a label so users know what type of file is required.
+When the File Input is used in a form, it should be grouped with related fields and include a label so users know what type of file is required.
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:92]: I think the form section here can be removed. 
I would add a section to best practices that says we always recommend including a label on an input.

File: src/_areas/design-system/components/file-inputs.md [hunk 7]
```diff
@@ -1,105 +1,133 @@
...
-![Single-file input icon](/images/components/file-input/singlefile_iconexamples.svg)
+![File Input form](/images/file-input_form.svg)
 
 ### Drag and drop
 
-When using the drag and drop functionality of a single-file input, the dropzone should be highlighted only when the user drags a file over the input. This will allow the user to differentiate between multiple inputs on a page and ensure that the correct file is being uploaded.
+When using the drag and drop functionality of a File Input, the dropzone should be highlighted only when the user drags a file over the input. This will allow the user to differentiate between multiple inputs on a page and ensure that the correct file is being uploaded.
 
-![Single-file input drag and drop](/images/components/file-input/singlefile_dropform.svg)
+![File Input drag and drop](/images/file-input_draganddrop.svg)
```
Comment (itsmemin) [inline src/_areas/design-system/components/file-inputs.md:102]: It would be excellent if we could get a GIF here to show the input highlighting.

Comment (honeckkaitlynn) [reply src/_areas/design-system/components/file-inputs.md:103]: Just tested and it looks like the drag and drop highlighting isn't implemented yet on our current file inputs.

**Pull request merged**